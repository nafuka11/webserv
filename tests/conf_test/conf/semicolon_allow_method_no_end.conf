server {
    server_name webserv01;
    listen 4242;

    allow_method GET
}