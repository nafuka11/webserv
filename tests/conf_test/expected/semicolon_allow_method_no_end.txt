webserv: directive "allow_method" has no end ";" in ./conf/semicolon_allow_method_no_end.conf:5
